src/vertex.cpp

```diff
@@ -9,8 +9,8 @@ Vertex::Vertex()
 	m_pos = Vec3();
 	m_normal = Vec3();
 	m_flags = VertexFlags::NONE;
-	m_colorHigh = Color(255u, 255u, 255u);
-	m_colorLow = Color(255u, 255u, 255u);
+	m_colorHigh = Color((unsigned char)255u, 255u, 255u);
```

Comment:
prefer using static_cast, also only need to cast if VS generates warnings, otherwise leave as is